content/2024/move-slow-and-fix-things/index.md

```diff
@@ -0,0 +1,113 @@
+
+That's a whole lot of risk for very little upside.
+
+You might wonder why I'm picking on Paul Graham so much.
+It's because I once looked up to him and valued his essays.
+He represents a worldview I used to believe in, but now consider harmful.
+Most of his essays seem true on the surface, but dig deeper and you'll find his claims are based on a narrow worldview and rarely supported by evidence.
+Misleading young, impressionable people is dangerous.
+
+## Is All VC Money Bad?
+
+Of course not. But I'd argue it's becoming less and less relevant in today's world.
+Take building a software product, for example. You don't need a fortune to get started anymore. There are website builders, cloud hosting solutions, and open source software at your fingertips.
+Why take on VC money when you're just starting out?
+Some might argue that you profit from valuable networking opportunities and business advice along with the funding.
+But most of that information is freely available online these days. There's an abundance of podcasts, videos, and books on the subject if you're willing to learn.
+
+It's trickier, of course, if you're building a physical product.
+But even that has become much easier in recent years.
+Could you sell a 3D printed prototype before scaling up production?
+Or launch a Kickstarter campaign to fund your first batch of products?
+There are now print-on-demand services for t-shirts, mugs, posters, and books.
+Plus, there are plenty of brick-and-mortar stores you could approach with your business idea if you're looking to collaborate.
+
+Let's say you do become wildly successful and urgently need capital to scale. Maybe then VC money is one way to go. Or you could take out a loan.
+Either way, remember that money always comes with strings attached. 
+Funding might force you to do things you're not comfortable with, like compromising your users' privacy or your own values.
+But often, if you spend enough time thinking through a problem, you might find a way to prove your concept at a smaller scale.
+
+## There Is No Infinite Growth
+
+Ask yourself: What truly motivates you? Is it power, money, or fame?
+If so, there are more meaningful things in life.
+Your self-worth isn't tied to building a unicorn. 
+It's perfectly fine to be 23 and not be a millionaire founder.
+If you're a high school or college student dreaming of running a startup, know that there's another path.
+
+Don't sell out your moral compass for a quick buck.
+You don't have to jump on the AI bandwagon just because it's the flavor of the month.
+[At least 144 out of 251 companies](https://www.ycombinator.com/companies?batch=W24&tags=Artificial%20Intelligence&tags=AI&tags=Generative%20AI&tags=AI%20Assistant) from the YC W24 batch are building products with "AI".
+That's 57% of the batch.
+How many of them are actually doing something meaningful with it?
+How many will still be around in 5 years? What lasting value do these companies bring to the world?
```

Comment:
Maybe you want to look into the past? How many of the W19 batch are still around, and how many of them did "blockchain"?

Reply:
As discussed, great idea, but hard to do. 😅